tutor/resources/styles/book-content/mixins.scss

```diff
@@ -83,6 +83,10 @@
 
     width: $tutor-interactive-iframe-width;
     margin-left: calc(50% - #{($tutor-interactive-iframe-width / 2)});
+
+    @if ($inExercise: true) {
+      padding: 0px 136px;
+    }
```

Comment:
@brittany-johnson I replaced the entire if block here with:
`    @include book-content-full-width(); `

and it looked pretty good to me:




<img width="1224" alt="image" src="https://user-images.githubusercontent.com/79566/54393694-cbc1b400-4678-11e9-89e7-3e329649a256.png">